sdk/typescript/src/utils/sui-types.ts

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) Mysten Labs, Inc.
```

Comment:
Curious why this file is named sui-types.

Reply:
open to suggestions, I don't have a better idea.  These are utils that were in the `types` directory before.  All the functions here are for things roughly related to validating/formatting of sui specific types (addresses, objects, etc)